### ISSUE TITLE
Compatibility with Bettercap 2.6 ?

### ISSUE THREAD

**Issue by nebukazar911** [v1s1t0r1sh3r3/airgeddon#147]
Hi,

Is there any plans to include compatibility with bettercap 2.6 (running under kali-rolling) ?
When launching an evil-twin attack with the bettercap/beef option, it seems to be running bettercap with the 1.6 arguments.

Thanks!

**Comment by v1s1t0r1sh3r3** [comment]
This is a duplicated of this other issue: https://github.com/v1s1t0r1sh3r3/airgeddon/issues/135

Compatibility with bettercap 2.x it's not planned. At least for now. Bettercap suffered major changes and it seems now it works in a very different way. The program does not even have an option to see the version. I wrote an email to the author but I had no response. It seems he doesn't care too much about the integration on scripts.

We already documented all on the wiki. It explains how to downgrade the package for some Linux distributions: https://github.com/v1s1t0r1sh3r3/airgeddon/wiki/Bettercap%20Tips

On new v8.10 version which is going to be released very soon now the new version is detected and it is shown a controlled error (a message with the downgrade suggestion).
